tests/migrations/test_operations.py

```diff
@@ -2953,3 +2956,57 @@ def test_reference_field_by_through_fields(self):
         self.assertIs(operation.references_field('Through', 'whatever'), False)
         self.assertIs(operation.references_field('Through', 'first'), True)
         self.assertIs(operation.references_field('Through', 'second'), True)
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'CreateIndexConcurrently is PostgreSQL-specific')
+class CreateIndexConcurrentlyTests(OperationTestBase):
```

Comment:
A note on these tests being in this file: I initially attempted to add a test file in the tests/postgres_tests directory, but I wanted to use some of the functionality provided by `OperationTestBase` and struggled with figuring out a clean way to import the class without causing errors. If this poses a problem, I am certainly willing to try again; I just might need some guidance.

Reply:
Update: I moved these after all. It did require me to move `OperationTestBase` into tests/migrations/test_base.py.